controller/users_blackbox_test.go

```diff
@@ -51,6 +54,76 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, NewUsersController(svc, s.GormDB, s.Configuration)
 }
 
+func (s *TestUsersSuite) TestDeleteUsers() {
+	t := s.T()
+	t.Run("ok", func(t *testing.T) {
+		// given
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.Identities(1), tf.Spaces(1))
```

Comment:
You must use `t` instead of `s.T()` because `s.T()` refers to the top level test and not the nested one.

Please double check all `s.T()` occurrences. There are many more here.

```suggestion
		fxt := tf.NewTestFixture(t, s.DB, tf.Identities(1), tf.Spaces(1))
```